easybuild/easyconfigs/s/snakemake/snakemake-5.26.1-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,83 @@
+easyblock = 'PythonBundle'
+
+name = 'snakemake'
+version = '5.26.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://snakemake.readthedocs.io'
+description = "The Snakemake workflow management system is a tool to create reproducible and scalable data analyses."
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('PyYAML', '5.3'),
+    ('GitPython', '3.1.9', versionsuffix),
+]
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('wrapt', '1.12.1', {
+        'checksums': ['b62ffa81fb85f4332a4f609cab4ac40709470da05643a082ec1eb88e6d9b97d7'],
+    }),
+    ('ratelimiter', '1.2.0.post0', {
+        'checksums': ['5c395dcabdbbde2e5178ef3f89b568a3066454a6ddc223b76473dac22f89b4f7'],
+    }),
+    ('ConfigArgParse', '1.2.3', {
+        'checksums': ['edd17be986d5c1ba2e307150b8e5f5107aba125f3574dddd02c85d5cdcfd37dc'],
+    }),
+    ('datrie', '0.8.2', {
+        'checksums': ['525b08f638d5cf6115df6ccd818e5a01298cd230b2dac91c8ff2e6499d18765d'],
+    }),
+    ('pyrsistent', '0.17.3', {
+        'checksums': ['2e636185d9eb976a18a8a8e96efce62f2905fea90041958d8cc2a189756ebf3e'],
+    }),
+    ('jsonschema', '3.2.0', {
+        'checksums': ['c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a'],
+    }),
+    ('traitlets', '5.0.5', {
+        'checksums': ['178f4ce988f69189f7e523337a3e11d91c786ded9360174a3d9ca83e79bc5396'],
+    }),
+    ('ipython_genutils', '0.2.0', {
+        'checksums': ['eb2e116e75ecef9d4d228fdc66af54269afa26ab4463042e33785b887c628ba8'],
+    }),
+    ('jupyter-core', '4.6.3', {
+        'modulename': 'jupyter_core',
+        'source_tmpl': 'jupyter_core-%(version)s.tar.gz',
+        'checksums': ['394fd5dd787e7c8861741880bdf8a00ce39f95de5d18e579c74b882522219e7e'],
+    }),
+    ('nbformat', '5.0.8', {
+        'checksums': ['f545b22138865bfbcc6b1ffe89ed5a2b8e2dc5d4fe876f2ca60d8e6f702a30f8'],
+    }),
+    ('toposort', '1.5', {
+        'checksums': ['dba5ae845296e3bf37b042c640870ffebcdeb8cd4df45adaa01d8c5476c557dd'],
+    }),
+    ('amply', '0.1.4', {
+        'checksums': ['cb12dcb49d16b168c02be128a1527ecde50211e4bd94af76ff4e67707f5a2d38'],
+    }),
+    ('PuLP', '2.3', {
+        'checksums': ['9d8ecf532868cc31fa9ff59ee5d5b2049600c5c902c18c794a2bad677c1f92e5'],
+    }),
+    (name, version, {
+        'checksums': ['94cfed2f47f6441cca6cf9d53cffdc11eb78dd1758d96c0af9cd5e453247363e'],
+    }),
+]
+
+# site-specific: the line below is the command for loading this module with the (default) EB module naming scheme
+# if necessary, change this command to match your cluster setup
+local_snakemake_load = 'module load %(name)s/%(version)s-%(toolchain_name)s-%(toolchain_version)s%(versionsuffix)s'
```

Comment:
@Darkless012 Not a blocker for this PR, but we should come up with a better approach for this, for example by adding support to EasyBuild framework for a new `%(mod_name)s` template value, so the active module naming scheme can be taken into account without having to tweak the easyconfig file.

Please open an issue on this in framework, and maybe try to take a look at implementing it?

Reply:
Quickly tackled that myself, see https://github.com/easybuilders/easybuild-framework/pull/3497

Reply:
Will take a look :) Thanks

Reply:
@boegel do you want to wait for this to incorporate the framework improvement?

Reply:
Yeah, https://github.com/easybuilders/easybuild-framework/pull/3497 is merged now, so we should leverage the new `%(module_name)s` template here.